### PR TITLE
Making test case to Fresnel function

### PR DESCRIPTION
**Pull request by nicoletraher**
5 test cases (corresponding to the 5 cases where the formula was changed).

#### Brief description of what is fixed or changed
Error in handling reciprocal square root in integral containing Fresnel functions #25093
Adding test cases

#### Other comments

NO ENTRY

### CODE REVIEW

**Review by nicoletraher — COMMENTED** [review]
Delete white space

**Comment by 1e9abhi1e10** [comment]
you don't need to create a new PR for adding the tests for #25093, you can add these tests in #25097.

**Review by nicoletraher — COMMENTED** [review]
Adding name

File: sympy/integrals/tests/test_integrals.py [hunk 1]
```diff
@@ -683,6 +683,16 @@ def test_integrate_returns_piecewise():
         Ne(x, y + 1)), (t, True))
 
 
+def test_manualintegrate_formula_change():
+    # Issue 25093
+    a = Symbol('a', positive=True)
```
Comment (smichr) [inline sympy/integrals/tests/test_integrals.py:688]: The `a` is not needed or else a test is missing.

**Comment by smichr** [comment]
Please join this with your other PR -- the tests for a change are usually included as a 2nd commit in the same branch.

**Pull request closed without merging**